pkg/apis/controller/experiments/v1alpha3/experiment_types.go

```diff
@@ -49,9 +49,11 @@ type ExperimentSpec struct {
 
 	NasConfig *NasConfig `json:"nasConfig,omitempty"`
 
+	// If false/true, which means delete/resume Suggestion after experiment is finished
+	ResumeExperiment bool `json:"resumeExperiment,omitempty"`
```

Comment:
Can we use this parameter in the future PRs also for https://github.com/kubeflow/katib/issues/1062?
For example:

`ResumeExperiment: "never"`- Always delete Suggestion deployment after Experiment is finished and never restart it.
`ResumeExperiment: "alwaysRunning"` - Suggestion deployment is in always running mode.
`ResumeExperiment: "fromPersistentVolume"` - Suggestion deployment is deleted after Experiment is finished and we can restore it from PV.

This is just my thought, we can think about better naming.
What do you think @sperlingxx ?
/cc @gaocegege @johnugeorge @richardsliu

Reply:
I think it is already updated.